docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -122,7 +120,7 @@ public class TokenSecuredResource {
         } else {
             name = ctx.getUserPrincipal().getName(); // <6>
         }
-        return String.format("hello + %s,"
+        return String.format("hello %s,"
```

Comment:
Without the plus is better but the quickstart still has it: https://github.com/quarkusio/quarkus-quickstarts/blob/main/security-jwt-quickstart/src/main/java/org/acme/security/jwt/TokenSecuredResource.java#L71.

I don't think it is significant, the presence or absence of `+` in what is some sample response

Reply:
@nkolatsis If you'd like, please open a PR for quickstarts to remove it